universal-login-web3/test/ULWeb3Provider.test.ts

```diff
@@ -15,12 +15,13 @@ chai.use(chaiAsPromised);
 describe('ULWeb3Provider', () => {
   let relayer: RelayerUnderTest;
   let deployer: Wallet;
-  let services: OnboardingProps;
+  let services: ULWeb3RootProps;
   let web3: Web3;
   let ulProvider: ULWeb3Provider;
 
   beforeEach(async () => {
     ({relayer, deployer, services, web3, ulProvider} = await setupTestEnvironmentWithWeb3());
+    sinon.replace((ulProvider as any).uiController, 'requireConfirmation', sinon.stub().resolves(true));
```

Comment:
I think this is more readable:
`ulProvider.requireConfirmation = sinon.stub().resolves(true)`